resources/openapi/yaml/api-observability.yaml

```diff
@@ -0,0 +1,102 @@
+openapi: 3.0.1
```

Comment:
i would remove the OpenApi stuff on this PR.
Eventual diff with main branch i guess we can solve it in a different PR wdyt?

Reply:
This was generate automatically actually.